charts/airbyte/values.yaml

```
@@ -92,6 +92,7 @@ webapp:
   service:
```

```suggestion
  ## @param webapp.service.annotations Annotations for the webapp service resource
  service:
```
The error correctly states that the field webapp.service.annotations is not annotated correctly (actually not at all). Add it like this.

Seems like this should run in a pre-commit hook in a github actions pipeline somewhere, to make sure we don't merge PRs with manually written text in the README like this because it will inevitably be overwritten by later PRs.

---

Thank @dingobar , I think we have can run a helm chart sanity check run in the CI to perform this check. Do you think running `./ci.sh check-docs-updated` would be enough? Feel free to open an issue on our repo to request this addition.